projects/observability/src/shared/graphql/request/handlers/entities/query/entity/entity-graphql-query-handler.service.ts

```diff
@@ -56,7 +56,9 @@ export class EntityGraphQlQueryHandlerService implements GraphQlQueryHandler<Gra
       limit: 1,
       timeRange: request.timeRange,
       properties: request.properties,
-      filters: [new GraphQlEntityFilter(request.id, request.entityType)]
+      filters: [new GraphQlEntityFilter(request.id, request.entityType)],
+      // If querying for a single API, then always want to includeInactive
+      includeInactive: request.entityType === ObservabilityEntityType.Api
```

Comment:
This would apply always right? Even if we disabled this functionality elsewhere

Reply:
Hmm. Good point. It would. Not sure how to feature flag this cleanly. Open to suggestions.